src/pages/ProgramProgressCollectionAdminPage/ProgramProcessBlock.tsx

```diff
@@ -181,10 +180,14 @@ const ProgramProcessBlock: React.VFC = () => {
                   memberEmail,
                   ...data.property.map(p => m.member_properties.find(mp => mp.property_id === p.id)?.value || ''),
                   Math.ceil(Number(watchedDuration) / 60).toString(),
-                  (watchedProgress * 100).toFixed(0) + '%',
+                  programContentType === 'exam' || programContentType === 'exercise'
+                    ? examProgressPercentage
+                    : (watchedProgress * 100).toFixed(0) + '%',
                   firstWatchedAt && dayjs(firstWatchedAt).tz(currentTimeZone).format('YYYY-MM-DD HH:mm:ss'),
                   lastWatchedAt && dayjs(lastWatchedAt).tz(currentTimeZone).format('YYYY-MM-DD HH:mm:ss'),
-                  ((memberWatchedDuration / (programDuration || 1)) * 100).toFixed(0) + '%',
+                  programContentType === 'exam' || programContentType === 'exercise'
+                    ? examProgressPercentage
+                    : ((memberWatchedDuration / (programDuration || 1)) * 100).toFixed(0) + '%',
```

Comment:
this column is totalPercentage
should be caculate by each program content body's type
if the body type is exam or exercise, decide by passing or not